README.md

```diff
@@ -1,10 +1,15 @@
-[![Build Status](https://travis-ci.com/spine-examples/kanban.svg?branch=master)](https://travis-ci.com/spine-examples/kanban) &nbsp;
-[![license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Build Status][travis-badge]](https://travis-ci.com/spine-examples/kanban) &nbsp;
+[![license][license-badge]](http://www.apache.org/licenses/LICENSE-2.0)
+
+[travis-badge]: https://travis-ci.com/spine-examples/kanban.svg?branch=master
+[license-badge]: https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat
 
 # Kanban Board example
 
 This example demonstrates:
-  * Multiple related Aggregates
-  * Orchestrating Aggregates using Process Managers.
-  * Building a Projection which is subscribed to both events and entity updates.
 
+* Multiple related Aggregate.
+  
```

Comment:
Why this change?

Reply:
Just looks a bit spacier for my liking. But, I guess in this case it's better to have them condensed.